src/InteractiveWindow/Editor/InteractiveWindow.UIThreadOnly.cs

```diff
@@ -2357,19 +2363,52 @@ private static NormalizedSnapshotSpanCollection GetSelectionSpans(ITextView text
                 return selection.SelectedSpans;
             }
 
+            /// <summary>
+            /// REPL clipboard format.
+            /// </summary>
+            private const string ReplFormat = "REPL";
```

Comment:
Should this be InteractiveCode or something along those lines?  We don't use REPL anywhere else